examples/webgl_modifier_subdivision.html

```diff
@@ -0,0 +1,250 @@
+
+					refreshDisplay();
+
+				} );
+
+				folder1.add( params, 'iterations' ).min( 0 ).max( 5 ).step( 1 ).onFinishChange( updateMeshes );
+				const splitController = folder1.add( params, 'split' ).onFinishChange( updateMeshes );
+				const smoothController = folder1.add( params, 'uvSmooth' ).onFinishChange( updateMeshes );
+				folder1.add( params, 'flatOnly' ).onFinishChange ( updateMeshes );
+				folder1.add( params, 'maxTriangles' ).onFinishChange( updateMeshes );
+
+				const folder2 = gui.addFolder( 'View' );
+				folder2.add( params, 'wireframe' ).onFinishChange( updateMeshes );
+
+				function refreshDisplay() {
+
+					geomController.updateDisplay();
+					splitController.updateDisplay();
+					smoothController.updateDisplay();
+
+					updateMeshes();
+
+				}
+
+			}
+
+			function getGeometry() {
+
+				switch ( params.geometry.toLowerCase() ) {
+
+					case 'box':
+						return new THREE.BoxGeometry();
+
+					case 'capsule':
+						return new THREE.CapsuleGeometry( 0.5, 0.5, 3, 5 );
+
+					case 'circle':
+						return new THREE.CircleGeometry( 0.6, 10 );
+
+					case 'cone':
+						return new THREE.ConeGeometry( 0.6, 1.5, 5, 5 );
+
+					case 'cylinder':
+						return new THREE.CylinderGeometry( 0.5, 0.5, 1, 5, 5 );
+
+					case 'dodecahedron':
+						return new THREE.DodecahedronGeometry( 0.6 );
+
+					case 'icosahedron':
+						return new THREE.IcosahedronGeometry( 0.6 );
+
+					case 'lathe':
+
+						// Sine Wave
+
+						const points = [];
+						for ( let i = 0; i < 65; i += 5 ) {
+							let x = ( Math.sin( i * 0.2 ) * Math.sin( i * 0.1 ) * 15 + 50 ) * 1.2;
+							let y = ( i - 5 ) * 3;
+							points.push( new THREE.Vector2( x * 0.0075, y * 0.005 ) );
+						}
+
+						const latheGeometry = new THREE.LatheGeometry( points, 4 );
+						latheGeometry.center();
+
+						return latheGeometry;
+
+					case 'octahedron':
+						return new THREE.OctahedronGeometry( 0.7 );
+
+					case 'plane':
+						return new THREE.PlaneGeometry();
+
+					case 'ring':
+						return new THREE.RingGeometry( 0.3, 0.6, 10 );
+
+					case 'sphere':
+						return new THREE.SphereGeometry( 0.6, 8, 4 );
+
+					case 'tetrahedron':
+						return new THREE.TetrahedronGeometry( 0.8 );
+
+					case 'torus':
+						return new THREE.TorusGeometry( 0.48, 0.24, 4, 6 );
+
+					case 'torusknot':
+						return new THREE.TorusKnotGeometry( 0.38, 0.18, 20, 4 );
+
+				}
+
+			}
+
+			function updateMeshes() {
+
+				meshNormal.geometry = getGeometry();
+
+				meshSmooth.geometry = LoopSubdivision.modify(
+					getGeometry(),
+					params.iterations,
+					params.split,
+					params.uvSmooth,
+					params.flatOnly,
+					params.maxTriangles
+				);
+
+				wireNormal.geometry = meshNormal.geometry.clone();
+				wireSmooth.geometry = meshSmooth.geometry.clone();
+
+				wireNormal.visible = params.wireframe;
+				wireSmooth.visible = params.wireframe;
+
+				wireNormal.scale.set( 1.01, 1.01, 1.01 );
+				wireSmooth.scale.set( 1.01, 1.01, 1.01 );
```

Comment:
See if https://stackoverflow.com/a/31541369/1461008 is sufficient to avoid this hack. If not, OK.

Reply:
Good call, I have changed the example to use `polygonOffset` instead of mesh scaling for the wireframe view.